server/tests/src/main/scala/sttp/tapir/server/tests/ServerMultipartTests.scala

```diff
@@ -15,19 +15,47 @@ import sttp.tapir.tests.Multipart.{
 import sttp.tapir.tests.TestUtil.{readFromFile, writeToFile}
 import sttp.tapir.tests.data.{FruitAmount, FruitData}
 import sttp.tapir.tests.{MultipleFileUpload, Test, data}
+import sttp.tapir.server.model.EndpointExtensions._
 
 import scala.concurrent.Await
 import scala.concurrent.duration.DurationInt
 
 class ServerMultipartTests[F[_], OPTIONS, ROUTE](
     createServerTest: CreateServerTest[F, Any, OPTIONS, ROUTE],
     partContentTypeHeaderSupport: Boolean = true,
-    partOtherHeaderSupport: Boolean = true
+    partOtherHeaderSupport: Boolean = true,
+    maxContentLengthSupport: Boolean = true
 )(implicit m: MonadError[F]) {
   import createServerTest._
 
   def tests(): List[Test] =
-    basicTests() ++ (if (partContentTypeHeaderSupport) contentTypeHeaderTests() else Nil)
+    basicTests() ++ (if (partContentTypeHeaderSupport) contentTypeHeaderTests() else Nil) ++
+      (if (maxContentLengthSupport) maxContentLengthTests() else Nil)
+
+  def maxContentLengthTests(): List[Test] = List(
+    testServer(in_simple_multipart_out_multipart.maxRequestBodyLength(314), "multipart with maxContentLength exceeded")((fa: FruitAmount) =>
```

Comment:
where does the 314 come from? :) an estimation of header length + boundaries + body?

maybe it would be better to test with much larger parts in the bodies, so that the tests are invulnerable to adding or removing 10 bytes from the headers, or if the boundaries happen to be longer

Reply:
Yeah, I estimated the test body with headers/boundaries. If I understand correctly, you propose to add bigger bodies and set `maxRequestBodyLength` to values that aren't exactly equal to body length, but higher, with some buffer for headers/boundaries?